src/qiboconnection/api.py

```diff
@@ -407,9 +403,14 @@ def execute(  # pylint: disable=too-many-locals, disable=too-many-branches
         job_ids = []
         logger.debug("Sending qibo circuits for a remote execution...")
         for job in jobs:
-            response, status_code = self._connection.send_post_auth_remote_api_call(
-                path=self._CIRCUITS_CALL_PATH, data=asdict(job.job_request)
-            )
+            try:
+                response, status_code = self._connection.send_post_auth_remote_api_call(
+                    path=self._CIRCUITS_CALL_PATH, data=asdict(job.job_request)
+                )
+            except HTTPError:  # delete ASAP
```

Comment:
This is a bit dirty: the request code is duplicated, it only retries once, there is no timeout between retries, etc.

How about implementing a "max_retries"? And some sleep() between replies (backoff_factor), so the connection could get fixed?

Reply:
oh, this is not installing retries for connectivity issues. We still dont have those at any part of qiboconnection sdk. This was intending to still work if the server was not updated to the new format.

But it is also not working, as the public part is digesting both new and old formats. Maybe I can implement some data validation to sent a client error if contents of description and results do not respect the new format in the newest structure.

Reply:
Ah, didn't see the difference, `job.job_request` != `job.job_request_deprecated`

Then it's good, no need for retries

Reply:
This was deleted in the end. We will take the approach of making the backend compatible with a couple of versions of qiboconnection, and not the other way arround.